ElementX/Sources/Services/Timeline/TimelineItemContent/TimelineItemReplyDetails.swift

```diff
@@ -19,6 +19,11 @@ import Foundation
 enum TimelineItemReplyDetails: Hashable {
     case notLoaded(eventID: String)
     case loading(eventID: String)
-    case loaded(sender: TimelineItemSender, contentType: EventBasedMessageTimelineItemContentType)
+    case loaded(sender: TimelineItemSender, repliedEventContent: TimelineEventContent)
```

Comment:
Given this is reply details, I don't think the `replied` provides any clarity here.
```suggestion
    case loaded(sender: TimelineItemSender, eventContent: TimelineEventContent)
```

Reply:
I like it!

Reply:
Applied https://github.com/vector-im/element-x-ios/pull/2087/commits/c70df2f3c5e5004016642a34b8f921a325bc97a2